### ISSUE TITLE
Is the validation set same as the training set?

### ISSUE THREAD

**Issue by Kalyan0821** [rhgao/co-separation#13]
Hi

Are the validation examples taken from the training data itself? I could not spot any differences in how you handle training vs validation data, except for the total number of examples used in each case. Because you just randomly sample a pair of videos in audioVisual_dataset.py, and the data loaders for training and validation use the same file paths, isn't it possible that some training videos are re-used for validation? 

Thanks for the great work.

**Comment by rhgao** [comment]
The training data and validation data are specified with different hdf5 file (which contains file paths). 
h5f_path = os.path.join(opt.hdf5_path, opt.mode+'.h5')

For training, opt.mode = 'train'
For validation, opt.mode = 'validation'

Training data and validation data are loaded using different data loaders.

**Comment by Kalyan0821** [comment]
Got it, thanks!
